### PR TITLE
Update CM4 recipe to initv3

### PR DESCRIPTION
**Pull request by foonerd**

### CODE REVIEW

File: recipes/devices/cm4.sh [hunk 1]
```diff
@@ -346,54 +403,79 @@ device_chroot_tweaks_pre() {
...
 	if [[ $DEBUG_IMAGE == yes ]]; then
-		log "Creating debug image" "wrn"
-		log "Adding Serial Debug parameters"
+		log "Creating debug image" "dbg"
+		log "Adding Serial Debug parameters" "dbg"
 		echo "include debug.txt" >>/boot/config.txt
 		cat <<-EOF >/boot/debug.txt
 			# Enable serial console for boot debugging
 			enable_uart=1
+			dtoverlay=pi3-miniuart-bt
```
Comment (Darmur) [inline recipes/devices/cm4.sh:467]: this should not be in the CM4 recipe

**Comment by Darmur** [comment]
please remove the "install-to-disk" logic from CM4 recipe.

CM4 recipe is for internal use, for generic use the standard PI image can be used with CM4

File: recipes/devices/cm4.sh (hunk 1 above)
Comment (foonerd) [reply recipes/devices/cm4.sh:467]: This should not be here: pi3-miniuart-bt

File: recipes/devices/cm4.sh [hunk 2]
```diff
@@ -47,12 +51,39 @@ PACKAGES=(
 	"wiringpi"	
 	# Wireless firmware
 	"firmware-atheros" "firmware-ralink" "firmware-realtek" "firmware-brcm80211"
+	# Install to disk tools
+	"liblzo2-2" "squashfs-tools"
 )
 
 ### Device customisation
 # Copy the device specific files (Image/DTS/etc..)
 write_device_files() {
```
Comment (Darmur) [inline recipes/devices/cm4.sh:60]: this should be removed too

Comment (Darmur) [reply recipes/devices/cm4.sh:60]: I mean the content of the function

Comment (foonerd) [reply recipes/devices/cm4.sh:60]: I see, this is part of the device build. Since tool is not referenced anymore - removing.

**Pull request merged**